Posting post-competition post (#147)
srobo/website: _posts/2019-04-07-sr2019-tlc-crowned-champion.md

---
layout: news
title: Team TLC Wins Student Robotics 2019
---

{% include figure.html

>            src="/images/content/news/sr2019/sr2019-photo.jpg"

Reviewer: This image is _huge_ -- about an order of magnitude larger than needed. Please can we resize it to be no larger than 1500px in either dimension. I would suggest that even 1000px wide is plenty.

The current image weight in this article is about 8.4M, which is a lot given that this article will be on our homepage for a considerable time.

If we want to keep the higher resolution images around and link to them that's fine, but the embeds should ideally be at the size they're going to be rendered at (or maybe 2x that if we want to bias towards high-dpi clients).

           caption="All the teams from SR2019" %}

That's a wrap on Student Robotics 2019! Our teams have been working hard since November to design, build, and test their robots; finally seeing the fruits of their work on the competition weekend. It was great to see our competitors quickly adapting their robots, from the physical design to the software, to overcome unforeseen challenges; giving themselves every small advantage towards claiming victory.

The Challenge: Caldera
----------------------

{% include figure.html
           class="right"
           src="/images/content/news/sr2019/arena.jpg"
           caption="The arena with a raised volcano area" %}

{% include figure.html
           class="right"
           src="/images/content/news/sr2019/tlc-launcher.jpg"
           caption="Team TLC's token launcher" %}

           
{% include figure.html
           class="right"
           src="/images/content/news/sr2019/swi.jpg"
           caption="Team SWI's volcano climbing robot in the Caldera" %}

> Our game this year, Caldera, challenged teams to control scoring zones with tokens while navigating an arena with raised platforms. Twenty-five scoring zones were arranged in a grid, varying in value from two points for each of the outermost 16 zones, a ring of eight on a raised “volcano” worth seven points apiece, to the the central “caldera” which was worth thirty points.

Reviewer: Should we also mention the robot-position bonus here?

I'm torn because it's a very necessary detail in order to understand both the match scores from the arena configurations, plus it indicates an orthogonal part of the challenge, however it's non-trivial to explain and I'm not sure it adds a lot to the high-level view of the game.

Author: "Parking your robot in a zone tripled the points of that zone and the 4 adjacent zones."

Is that explained simply enough?

> There were a few designs patterns this year. Many teams opted for tank tracks as a method of climbing the volcano. Instead of climbing the volcano, a few teams built catapult mechanisms with varying degrees of success. Others scooped up tokens and deposited them on the low scoring areas and/or the edge of the volcano. Some teams lifted tokens onto a top-mounted conveyor belts which dropped them onto the volcano's surface.

Reviewer 2: ```suggestion
There were a range of inventive designs this year. Many teams opted for tank tracks as a method of climbing the volcano. Instead of climbing the volcano, a few teams built catapult mechanisms with varying degrees of success. Others scooped up tokens and deposited them on the low scoring areas and/or the edge of the volcano. Some teams lifted tokens onto top-mounted conveyor belts which dropped them onto the volcano's surface.
```

> The most notable designs were:

Reviewer 3: ```suggestion
Some of the more notable designs were:
```

> - Team TLC with their consistent, successful catapult.

Reviewer: The TLAs won't mean much to external readers, I suggest we use the school names (optionally with the team's own names too) instead.

Perhaps have a look at the previous post-competition posts to see how we've done this in the past?

I suspect something like "$team_name (from $school_name)" for the case with a team name and "$school_name" or "The team from $school_name" for the case without is typical.

There might be value in working the TLAs in there somehow too, so that if the readers do click through to the scores then those pages make sense, but I think using the names is the priority.

- Team SWI with their simple and effective grab-and-go robot which climbed the volcano with a single token.
- Team CRB with their suction cup and catapult combination which unfortunately had technical issues for much of the competition.
- Team HAB with their Hawaiian themed robot, topped with a 3D paper pineapple.

The Final
---------

> Student Robotics 2019 concluded with an intense match between PSC, HAM, TLC, and HRS. Team HAM were quick off the mark pushing their tokens into the scoring zones. Team HRS soon followed, violently shoving their tokens across the arena. Team PSC's robot slowly moved 4 of their tokens into a single scoring zone before unfortunately flipping onto its back. Meanwhile, Team TLC quietly moved towards the edge of the volcano and launched a token directly into the Caldera.

Reviewer: Since we're a non-violent competition (and repeatedly need to encourage people not to compare us to robot wards), perhaps use a word other than "violently" here?
I didn't actually see the match, so I don't know how appropriate the description is, however if it was truly violent then I'd question why they weren't disqualified.

<!-- Video of Final -->
<iframe
  class="center video"
  src="https://www.youtube.com/embed/gOeP96OngE0"
  frameborder="0"
  width="100%"
  allowfullscreen
  >
</iframe>

We saw some brilliant robot designs this year, great work by all teams. We hope you enjoyed the competition as much as we did!

If you didn't take part, or you want to enter again next year, the [sign up page](/schools/how_to_enter) for next year's competition will be up later in the year. Get a team together and start talking to your teachers now!

Results
-------

> |        Prize          |            Team

Reviewer: for info (no need to change it): I don't think the tables need the preceding `|` character.

|-----------------------|-----------------------------------------------
| 1st Place             | [The Ladies' College](http://www.ladiescollege.com/)
| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)

> | Rookie Award          | [Eltham College](https://www.elthamcollege.london/)

Reviewer: Just a thought -- could we use title text on the names of the prizes to include the descriptions of the awards in the page?

I suspect that something like the following would do this:
``` markdown
<span title="The team which eats the most chocolate">Award name</span> | Team Name Here
```

| Committee Award       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
| Robot and Team Image  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
| Online Presence       | [Collyer's](https://www.collyers.ac.uk/) ([Twitter](https://twitter.com/CollyersRobots))

> For full details on the awards, please see the [rulebook](/docs/resources/2019/rulebook.pdf).

Reviewer: I think we should try to include more detail here about the awards. The non-place awards are fairly important -- the competition is about the competitors learning as much as it is about the robots doing well, so the other prizes help encourage that.

At a minimum, please could we include something about why SWI won the committee award? We already have the spiel for that, so I'm hoping that doing so is easy.

> Of course, this competition would not be possible without the support of [Motorola](https://www.motorolasolutions.com/en_us/about/company-overview/corporate-responsibility/motorola-solutions-foundation.html), our sponsor for this year, who enabled us to provide a fun and educational opportunity for students from a range of engineering disciplines. 

Reviewer 3: ```suggestion
Of course, this competition would not be possible without the support of [The Motorola Foundation](https://www.motorolasolutions.com/en_us/about/company-overview/corporate-responsibility/motorola-solutions-foundation.html), our major sponsor for this year, who enabled us to provide a fun and educational opportunity for students from a range of engineering disciplines. 
```

Reviewer 2: Motorola Solutions Foundation.

<!--Media
-----

> Photos can be found on the Student Robotics 2019 Competition [Flickr group](http://www.flickr.com/groups/sr2019/).-->

Reviewer 3: Are we not releasing these?

Author: I'm not sure what the plan is this year for publishing photos. I believe there was some suggestion that we don't use Flickr for GDPR reasons?

Reviewer: There is also the reality that we want this post to go live today (Monday) or tomorrow, but don't expect that the media will be present there yet, nor does that group even yet exist.

See also https://github.com/srobo/tasks/issues/277.

Reviewer 4: My suggestion is to remove this link until we've sorted https://github.com/srobo/tasks/issues/277. Flickr recently changed their pricing policy, making it less ideal for our uses, so i'll have to do some more digging into how to distribute these.

> Photos can be found on the Student Robotics 2019 Competition [Flickr group](http://www.flickr.com/groups/sr2019/).-->

Reviewer: nit: please can we use an HTTPS link here.

_The SR Team_

> 

Reviewer: It might be worth including a "Notes to editors" section here, like the one on the [SR2016 post](https://studentrobotics.org/news/2016-05-22-rgs-victorious-sr2016/). We could likely copy/paste that one, though some parts will need tweaking:
- end of the first paragraph ("more than 50 teams and over 400 students") 
- comment about the timing of the competition ("end of the Easter holiday")
- link to the flickr group (maybe remove if you're sharing the link above?)
